tsconfig.json

```diff
@@ -13,5 +13,6 @@
     ".eslintrc.js",
     "babel.config.js",
     "react-native.config.js",
+    "docsite",
```

Comment:
is fine for now, but we might want to be inclusive rather than exclusive, i.e. just src and examples/src. But no need to address